packages/zapp/console/src/components/LaunchPlan/SearchableLaunchPlanNameList.tsx

```diff
@@ -0,0 +1,169 @@
+import { debounce } from 'lodash';
+import { Typography, FormGroup } from '@material-ui/core';
+import { ResourceType } from 'models/Common/types';
+import { MuiLaunchPlanIcon } from '@flyteconsole/ui-atoms';
+import { LaunchPlanListStructureItem } from './types';
+import { SearchableInput } from '../common/SearchableList';
+import { useSearchableListState } from '../common/useSearchableListState';
+import t, { patternKey } from '../Entities/strings';
+import { entityStrings } from '../Entities/constants';
+
+interface SearchableLaunchPlanNameItemProps {
+  item: LaunchPlanListStructureItem;
+}
+
+interface SearchableLaunchPlanNameListProps {
+  launchPlans: LaunchPlanListStructureItem[];
+}
+
+export const showOnHoverClass = 'showOnHover';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  container: {
+    padding: theme.spacing(2),
+    paddingRight: theme.spacing(5),
+  },
+  filterGroup: {
+    display: 'flex',
+    flexWrap: 'nowrap',
+    flexDirection: 'row',
+    margin: theme.spacing(4, 5, 0, 2),
+  },
+  itemContainer: {
+    padding: theme.spacing(3, 3),
+    border: 'none',
+    borderTop: `1px solid ${separatorColor}`,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    position: 'relative',
+    // All children using the showOnHover class will be hidden until
+    // the mouse enters the container
+    [`& .${showOnHoverClass}`]: {
+      opacity: 0,
+    },
+    [`&:hover .${showOnHoverClass}`]: {
+      opacity: 1,
+    },
+  },
+  itemName: {
+    display: 'flex',
+    fontWeight: 600,
+    color: primaryTextColor,
+    alignItems: 'center',
+  },
+  itemDescriptionRow: {
+    color: '#757575',
+    marginBottom: theme.spacing(2),
+    width: '100%',
+  },
+  itemIcon: {
+    marginRight: theme.spacing(2),
+    color: '#636379',
+  },
+  itemRow: {
+    display: 'flex',
+    marginBottom: theme.spacing(1),
+    '&:last-child': {
+      marginBottom: 0,
+    },
+    alignItems: 'center',
+    width: '100%',
+  },
+  itemLabel: {
+    width: 140,
+    fontSize: 14,
+    color: launchPlanLabelColor,
+  },
+  searchInputContainer: {
+    padding: 0,
+  },
+}));
+
+/**
+ * Renders individual searchable launchPlan item
+ * @param item
+ * @returns
+ */
+const SearchableLaunchPlanNameItem: React.FC<SearchableLaunchPlanNameItemProps> = React.memo(
+  ({ item }) => {
+    const commonStyles = useCommonStyles();
+    const listStyles = useNamedEntityListStyles();
+    const styles = useStyles();
+    const { id, description } = item;
+
+    return (
+      <Link
+        className={commonStyles.linkUnstyled}
+        to={Routes.LaunchPlanDetails.makeUrl(id.project, id.domain, id.name)}
+      >
+        <div className={classNames(listStyles.searchResult, styles.itemContainer)}>
+          <div className={styles.itemName}>
+            <MuiLaunchPlanIcon width={16} height={16} />
+            <div>{id.name}</div>
+          </div>
+          {description && (
```

Comment:
nit: we currently never have a description in this items. Should we remove it and related styles from code?

Reply:
fixed